interfaces/kmod/kmod.go

```diff
@@ -18,3 +18,61 @@
  */
 
 package kmod
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/snap"
+)
+
+func moduleFilePath(snapName string) string {
+	confFileName := fmt.Sprintf("%s.conf", snap.SecurityTag(snapName))
```

Comment:
Great! I in fact prefer this approach (I even mentioned it as an option in the initial desgn review :) since it mirrors what we do everywhere else wrt to sandboxing and interface connections.